src/nvim/tui/tui.c

```diff
@@ -1268,7 +1268,7 @@ static void patch_terminfo_bugs(TUIData *data, const char *term,
   bool teraterm = terminfo_is_term_family(term, "teraterm");
   bool putty = terminfo_is_term_family(term, "putty");
   bool screen = terminfo_is_term_family(term, "screen");
-  bool tmux = terminfo_is_term_family(term, "tmux");
+  bool tmux = terminfo_is_term_family(term, "tmux") || !!os_getenv("TMUX");
```

Comment:
It was suggested to me that it would be more robust to detect tmux if we used that.
There are other such variables for VTE and konsole. So I don't how much potential hacks we should try to prevent.